### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by SerhiiiOselskyi**
- [DEMO LINK](https://bran4u.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://bran4u.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+body {
+  padding-top: 12px;
+}
+
+.search-bar-small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  font-style: normal;
```
Comment (irenhh) [inline src/style.css:19]: are you sure you need font-style here?

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+.search-bar-big {
+  height: 70px ;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background: #fff;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
```
Comment (irenhh) [inline src/style.css:35]: those are repeated styles

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+.search-bar-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  background-size: 19px;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
```
Comment (irenhh) [inline src/style.css:72]: there is no need to add the same styles to focused state, it inherits all styles from normal state

**Comment by irenhh** [comment]
image

**Comment by bran4u** [comment]
image
? Не могу повторить.

**Comment by irenhh** [comment]
this is mockup
image

this is yours
image

are fonts the same?

**Comment by danheim** [comment]
Не соответствует макету. 
Отсуствует hover эффект, аутлайн при фокусе некорректный.

**Pull request closed without merging**